.rubocop.yml

```diff
@@ -3,3 +3,4 @@ Metrics/LineLength:
 
 Metrics/BlockLength:
   Max: 100
+
```

Comment:
@MWilsonPuppet, do you want to remove this extra newline? After that, this looks good to me.